lib/stat_tracker.rb

```diff
@@ -37,43 +37,153 @@ def lowest_total_score
 
   def percentage_home_wins
     home_wins = 0
-    away_wins = 0
+    visitor_wins = 0
+    ties = 0
     CSV.foreach(game_teams, headers: true, header_converters: :symbol) do |row|
-      next if row [:result] == "LOSS"
-      if row[:hoa] == "away"
-        away_wins += 1
+      next if row[:result] == "LOSS"
+      if row[:result] == "TIE"
+        ties += 0.5
+      elsif row[:hoa] == "away"
+        visitor_wins += 1
       elsif row[:hoa] == "home"
         home_wins += 1
       end
     end
-    total_games = home_wins + away_wins
+    total_games = home_wins + visitor_wins + ties
     percentage = calc_percentage(home_wins, total_games)
   end
 
-  def percentage_away_wins
+  def percentage_visitor_wins
     home_wins = 0
-    away_wins = 0
+    visitor_wins = 0
+    ties = 0
     CSV.foreach(game_teams, headers: true, header_converters: :symbol) do |row|
-      next if row [:result] == "LOSS"
-      if row[:hoa] == "away"
-        away_wins += 1
+      next if row[:result] == "LOSS"
+      if row[:result] == "TIE"
+        ties += 0.5
+      elsif row[:hoa] == "away"
+        visitor_wins += 1
       elsif row[:hoa] == "home"
         home_wins += 1
       end
     end
-    total_games = home_wins + away_wins
-    percentage = calc_percentage(away_wins, total_games)
+    total_games = home_wins + visitor_wins + ties
+    percentage = calc_percentage(visitor_wins, total_games)
+  end
```

Comment:
Extremely readable, good that we agreed on breaking into components until we refactor together!